.github/workflows/phpcs.yml

```
@@ -0,0 +1,32 @@
+name: phpcs
+
+on:
+  pull_request:
+    branches:
+      - trunk
+      - develop
+      - develop-v1
+
+jobs:
+  changed:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - id: changed-files
+        uses: tj-actions/changed-files@v19
```

```suggestion
        uses: tj-actions/changed-files@v36
```

Support suggests this resolves the deprecation warning https://github.com/tj-actions/changed-files/issues/1206